File: examples/nodejs/basic_usage.js
#!/usr/bin/env node
/**
 * Basic usage example for HiTechCloud Nodejs SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_API_KEY=your-api-key
 */
const HiTechCloud = require('@hitechcloud/sdk');

const client = new HiTechCloud({ apiKey: 'your-api-key' });

// List all services
const services = await client.services.listServices();
console.log(services.data);

// Get a specific service
const service = await client.services.getService({ id: 123 });
console.log(service);

// Power operations
await client.services.startService({ id: 123 });
await client.services.stopService({ id: 123 });
await client.services.rebootService({ id: 123 });

File: examples/java/basic_usage.java
#!/usr/bin/env java
/**
 * Basic usage example for HiTechCloud Java SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_API_KEY=your-api-key
 */
import vn.hitechcloud.HiTechCloud;

HiTechCloud client = new HiTechCloud("your-api-key");

// List all services
var services = client.services().listServices();
System.out.println(services);

// Get a specific service
var service = client.services().getService(123);
System.out.println(service);

// Power operations
client.services().startService(123);
client.services().stopService(123);
client.services().rebootService(123);
